packages/nimble-components/src/select/index.ts

```diff
@@ -837,6 +931,41 @@ export class Select
         this.committedSelectedOption = options[this.selectedIndex];
     }
 
+    private toggleNewActiveOption(newActiveIndex: number): void {
```

Comment:
Perhaps this should be called `updateNewActiveOption` since it isn't actually "toggling" the option.

Reply:
Renamed to `setActionOption`.